packages/core/strapi/lib/core/bootstrap.js

```diff
@@ -22,4 +23,7 @@ module.exports = function({ strapi }) {
   if (!shouldServeAdmin) {
     strapi.config.serveAdminPanel = false;
   }
+
+  // ensure public repository exists
+  fse.ensureDir(strapi.dirs.public);
```

Comment:
Should we throw an error if it doesn't exist in prod? Feels like we will have the same read only problem otherwise 🤔